app/views/projects/_form.html.erb

```diff
@@ -65,13 +65,20 @@
   <div class="btn-group actions pull-right clearfix">
     <%= f.submit nil, class: 'btn btn-primary btn-form' %>
 
-    <% if policy(project).archive? %>
+    <% if policy(project).archive? && !project.archived %>
       <%= link_to t('projects.archive'), [:archive, project],
         data: {confirm: t('projects.index.are you sure you want to archive this project')},
         method: :patch,
         class: 'btn btn-warning btn-form' %>
     <% end %>
 
+    <% if policy(project).unarchive? && project.archived %>
+      <%= link_to t('projects.unarchive'), [:unarchive, project],
+        data: {confirm: t('projects.index.are you sure you want to unarchive this project')},
```

Comment:
`t('projects.unarchive_message')` if is not used in index os is used in more than one place, makes no sense keep under :index, and use shorter keys instead the text of the translation keeping the underscore pattern for key naming